### PR TITLE
Fixing some issues with the data connect docs

### PR DESCRIPTION
**Pull request by tlenig**
Received some feedback that our links are broken in the get-started section and fixing the name of the ToC

### CODE REVIEW

**Comment by Lauragra** [comment]
@tlenig , FYI, I already fixed the links. You can verify here: https://review.docs.microsoft.com/en-us/graph/data-connect-get-started?branch=master. This fix will go live today.

File: concepts/toc.yml [hunk 1]
```diff
@@ -90,7 +90,7 @@
     - name: Customer booking (preview)
       href: booking-concept-overview.md
       displayName: Bookings, Microsoft Bookings
-    - name: Data access
+    - name: Data connect
```
Comment (Lauragra) [inline concepts/toc.yml]: FYI, we're leaving the title of the node as "Data access". Data connect is the only child node right now, but we'll be adding docs for connectors here too, so we'll have two peer nodes.

Comment (tlenig) [reply concepts/toc.yml]: Ah interesting, are we concerned about putting both data connect and connectors in the same section of our docs? The names of the products are already so close, associating them in the same section might make that even more confusing, at least from my perspective

Comment (Lauragra) [reply concepts/toc.yml]: Yes, the naming similarity is a bridge that we will need to cross when we get there. ;) Note that in the Learn section of the TOC, we don't use product or feature names, we lead with the capability, so Data access is more consistent with that approach.

Comment (tlenig) [reply concepts/toc.yml]: Understood makes sense thanks!

**Pull request closed without merging**